colin-api/src/colin_api/models/filing.py

```diff
@@ -917,6 +919,9 @@ def get_filing(cls, filing: Filing, con=None, year: int = None) -> Dict:
                 else:
                     components = schema['properties'][filing.filing_type].get('properties').keys()
 
+            if filing_event_info['filing_type_code'] == 'CO_DI':
```

Comment:
What about including `AM_DI`? Same for line 974.

Reply:
These changes are limited to the filings filed after the alteration of BEN to BC on the 97 businesses. As of now AM_DI is not filed in any of those business

Reply:
k, good to me.